bin/docserv

```diff
@@ -141,11 +147,12 @@ class Deliverable:
             cmd = shlex.split(commands[i]['cmd'])
             logger.debug("Thread %i: %s" % (thread_id, commands[i]['cmd']))
             time.sleep(1)
-            #s = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-            #s.communicate()[0]
-            #if commands[i]['ret_val'] is not None and not commands[i]['ret_val'] == int(s.returncode):
-            #    logger.warning("Build failed! Unexpected return value %i for '%s'" % (s.returncode, commands[i]['cmd']) )
-            #    return False
+            s = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+            s.communicate()[0]
+            if commands[i]['ret_val'] is not None and not commands[i]['ret_val'] == int(s.returncode):
```

Comment:
I think, the `int()` in `s.returncode` is useless, as it is already an integer, right?